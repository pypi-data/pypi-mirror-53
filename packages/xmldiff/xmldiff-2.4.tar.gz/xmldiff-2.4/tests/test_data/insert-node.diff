[update-text, /body/div[1], "\n    "]
[insert, /body/div[1], p, 0]
[update-text, /body/div/p[1], "Simple text"]
[update-text-after, /body/div/p[1], "\n  "]
