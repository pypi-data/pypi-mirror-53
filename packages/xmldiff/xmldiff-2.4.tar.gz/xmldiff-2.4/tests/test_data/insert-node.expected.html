<body xmlns:diff="http://namespaces.shoobx.com/diff">
  <div id="id">
    <p diff:insert="">Simple text</p>
  </div>
</body>
